evm_body.yaml

```diff
@@ -256,15 +257,18 @@ eth_sendPrivateTransaction:
               tx:
                 type: string
                 description: 'Raw, signed transaction'
+                default: '0x'
```

Comment:
hmm will this work?

Reply:
Great catch, this will not work. 

Will fix this! 

Thank you for pointing this out.

Reply:
we can't ever get this to work since it involves sending a tx but we should at least put a valid signed tx string in there